### PR TITLE
Loan Policy Page - fixed incorrect behavior

### PR DESCRIPTION
**Pull request by maximdidenkoepam**
# Purpose
To fix Loan Policy Form bugs and current behavior.

# Link
https://issues.folio.org/browse/UICIRC-121
https://issues.folio.org/browse/UICIRC-136
https://issues.folio.org/browse/UICIRC-137

# Screenshots
screen shot 2018-12-24 at 18 36 05

### CODE REVIEW

**Comment by maximdidenkoepam** [comment]
Please stop to review it and wait until my confirmation. The story is blocked.

**Comment by maximdidenkoepam** [comment]
PR is available for review. @VictorSoroka1 @OleksandrAntonenko1

File: settings/LoanPolicy/components/DeleteEntity/DeleteEntity.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+  render() {
+    const {
+      confirmDelete,
+      policyName,
+    } = this.props;
+
+    const confirmationMessage = <FormattedMessage
+      id="ui-circulation.settings.loanPolicy.deleteMessage"
+      values={{ name: <strong>{policyName}</strong>, removed: <strong>removed</strong> }}
```
Comment (doytch) [inline settings/LoanPolicy/components/DeleteEntity/DeleteEntity.js]: "removed" won't be translated if we write it this way. Use `<SafeHTMLMessage>` instead, like in EntryForm.js

**Comment by julianladisch** [comment]
@maximdidenkoepam : Have you signed the CLA for folio-org/ui-circulation? Open Library Foundation does not accept code contributions unless the contributor signs the CLA to ensure that the contributor explicitly licenses the code under the terms of the Apache License Version 2.0.

File: settings/LoanPolicy/components/DeleteEntity/DeleteEntity.js (hunk 1 above)
Comment (OleksandrAntonenko1) [reply settings/LoanPolicy/components/DeleteEntity/DeleteEntity.js]: I have fixed that. Thank you.

**Comment by maximdidenkoepam** [comment]
> @maximdidenkoepam the code looks good to me but I think what @aditya-matukumalli mentioned above has to be addressed before we can merge this in.

@mkuklis necessary changes already in PR - https://github.com/folio-org/platform-core/pull/151/files

**Comment by maximdidenkoepam** [comment]
@aditya-matukumalli this is expected behavior(discussed with Sean Thomas (ui-circulation Product Owner)), to make this field editable you should pick interval first. In this PR - https://github.com/folio-org/platform-core/pull/151/files you can find fixed tests.

**Pull request merged**